x-pack/plugin/security/qa/multi-cluster/src/javaRestTest/java/org/elasticsearch/xpack/remotecluster/RemoteClusterSecurityEsqlIT.java

```diff
@@ -259,7 +262,7 @@ public void populateData() throws Exception {
               "remote_indices": [
                 {
                   "names": ["employees"],
-                  "privileges": ["read", "read_cross_cluster"],
+                  "privileges": ["read"],
                   "clusters": ["my_remote_cluster"]
```

Comment:
nit: no need to add it in this PR: but another test cases we should add for FLS and DLS

Reply:
DLS test add on https://github.com/elastic/elasticsearch/pull/107079/commits/a893be9b52d86fbddebdb8c60d7543cfffb20f87

Reply:
FLS added on https://github.com/elastic/elasticsearch/pull/107079/commits/6d1eef75264b0e9a545783476c7e240e76259096